openmetadata-ui/src/main/resources/ui/cypress/common/common.js

```diff
@@ -189,7 +189,7 @@ export const testServiceCreationAndIngestion = (
   );
   cy.get('[data-testid="next-button"]').should('exist').click();
   verifyResponseStatusCode('@ingestionPipelineStatus', 200);
-  verifyResponseStatusCode('@ipApi', 204);
+  cy.wait('@ipApi');
```

Comment:
Why we removed status code here?

Reply:
locally it comes as 204, and in CI it's 200, both are not sync

Reply:
I think we haven't changed it's value to other yaml files.
Can you check with @pmbrull & @akash-jain-10 once
It should be same as locally it's a bug